src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -33,7 +33,7 @@ public void Initialize(EntityInfo entity, Func<IncomingMessageDetails, ReceiveCo
             receiveMode = settings.Get<ReceiveMode>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.ReceiveMode);
 
             incomingCallback = callback;
-            this.errorCallback = errorCallback;
+            this.errorCallback = errorCallback ?? EmptyErrorCallback;
```

Comment:
Do we need this anymore? The `errorCallback` is now [passed](https://github.com/Particular/NServiceBus.AzureServiceBus/blob/101746ffbaf7d1869b8e08ffd110f691b4f317ca/src/Transport/Seam/MessagePump.cs#L69) from the message pump into topology operator. And topology operator [passes](https://github.com/Particular/NServiceBus.AzureServiceBus/blob/101746ffbaf7d1869b8e08ffd110f691b4f317ca/src/Transport/Topology/TopologyOperator.cs#L85) the callback into message receive notirifer. Therefore the callback cannot be null.